### ISSUE TITLE
[A11y] Country selector is not accessible by keyboard

### ISSUE THREAD

**Issue by meshe03** [catamphetamine/react-phone-number-input#373]
Hi! In my current project we are using this library. We want to make our web more accessible. I checked the demo and the select is focused as expected but when enter key is pressed the country selector isn't triggered. This implies that there is no way to modify the country for people who only navigate with the keyboard.

![a11y](https://user-images.githubusercontent.com/5552376/98151025-5094bd80-1eae-11eb-85a3-ac8f4aa5e3c7.gif)

**Comment by catamphetamine** [comment]
Hi. The reason for that is that the country selector uses a native `<select/>`. The native `<select/>` doesn't expand on `Enter`: it only expands on spacebar. I'm on Windows and this is how it works in this OS. I assume you're on a Mac, in which case that would mean it's an officially standardized behavior. 

We could add a hack: something like `<select onKeyDown={ if Enter then select.click() }/>` but why would we if that is not how a standard select works, and a standard select can't misbehave.

I can see there're upvotes on your comment. I suppose my answer is correct.

**Comment by catamphetamine** [comment]
Actually, I should test it first.

**Comment by catamphetamine** [comment]
Actually, it does expand on Enter on the demo page for me.
I'm on Windows.
Is this a Mac-specific issue?

**Comment by catamphetamine** [comment]
I'm on Chrome, btw.

Tested in Firefox, and in Firefox it doesn't expand on Enter.
So, this behavior is standardized and is browser-specific.
So, it's not an issue.

**Comment by meshe03** [comment]
Oh I see! I tried with space bar in my Mac using chrome and it works perfectly. And you're right, this is a standard behavior on select so it doesn't make much sense to force the list to be displayed by doing enter.

Thank you so much for your reply 😄
